flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/SlotSharingExecutionSlotAllocator.java

```diff
@@ -117,8 +147,7 @@ class SlotSharingExecutionSlotAllocator implements ExecutionSlotAllocator {
      *
      * @param executionVertexIds Execution vertices to allocate slots for
      */
-    @Override
-    public List<SlotExecutionVertexAssignment> allocateSlotsFor(
+    private List<SlotExecutionVertexAssignment> allocateSlotsForVertices(
```

Comment:
Currently, the `SlotExecutionVertexAssignment` is only used in `SlotSharingExecutionSlotAllocator`, maybe let it as an inner class of `SlotSharingExecutionSlotAllocator` .

Reply:
Agreed.